test/client.js

```diff
@@ -39,7 +39,7 @@ describe('http', () => {
       }
 
       fs.exists(filename, function (exists) {
-        if (exists) {
+        if (exists && filename[filename.length - 1] !== "/") {
```

Comment:
I removed this and all tests continued to pass.

I think this was a transient issue related to the failing XMock test.